src/loader/loader.c

```diff
@@ -461,9 +474,11 @@ post_analyze_hook(ParseState *pstate, Query *query, JumbleState *jstate)
 				break;
 		}
 	}
-	if (!guc_disable_load &&
-		(query->commandType != CMD_UTILITY || load_utility_cmd(query->utilityStmt)))
-		extension_check();
+	for (size_t i = 0; i < sizeof extensions / sizeof(struct ext); ++i) {
+		struct ext *const ext = &extensions[i];
+	if (!ext->guc_disable_load &&
+		(query->commandType != CMD_UTILITY || load_utility_cmd(query->utilityStmt, ext)))
+		extension_check(ext);
```

Comment:
I think there's a formatting bug here

Reply:
I'm avoiding any reindenting until the end, for ease of diffing.